synapse/events/third_party_rules.py

```diff
@@ -137,18 +258,17 @@ async def check_visibility_can_be_modified(
         Returns:
```

Comment:
Could you mention in the docstring that `False` will be returned as long as one `ThirdPartyRulesModule` denies the request?

Reply:
This is more following the rule of thumb of running several modules, so the right fix would be adding more documentation in `modules.md`, I'll do that in a separate PR.